libcontainer/process_linux.go

```diff
@@ -178,6 +179,49 @@ func (p *setnsProcess) start() (retErr error) {
 		case procHooks:
 			// This shouldn't happen.
 			panic("unexpected procHooks in setns")
+		case procSeccomp:
+			// receive seccomp-fd
+			pidfd, _, err := unix.Syscall(unix.SYS_PIDFD_OPEN, uintptr(p.pid()), 0, 0)
+			if err != 0 {
+				return newSystemErrorWithCause(err, "performing SYS_PIDFD_OPEN syscall")
+			}
+			defer unix.Close(int(pidfd))
+
+			seccompFd, _, err := unix.Syscall(unix.SYS_PIDFD_GETFD, pidfd, uintptr(sync.Fd), 0)
+			if err != 0 {
+				return newSystemErrorWithCause(err, "performing SYS_PIDFD_GETFD syscall")
+			}
+			defer unix.Close(int(seccompFd))
+
+			if p.config.Config.Seccomp.ListenerPath == "" {
```

Comment:
Maybe makes sense to move this check up (no sense in calling PIDFD_OPEN and PIDFD_GETFD if the path is not set)?

Reply:
Good point, moving it!

Reply:
FYI, this was moved to the top. Just that github is smart enough to move the comment to the top too :)